### PR TITLE
Feature/1201480035349749 document environment setup for mvp 2

### PR DESCRIPTION
**Pull request by irediaes**
* Added rust API lib README
* Updated App README

### CODE REVIEW

File: rust/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+
+# Peaq Network CharmEV Rust API Library
+
+#### Introduction
+This library handles the core api features of:
+* interacting with the node
+* connecting to provider peer network
+* managing and sending p2p events to the app business logic
+* generating consumer keys during onboarding
+* fetching and and parsing DID Document.
```
Comment (faisalthaheem) [inline rust/README.md]: repetition in and

File: rust/README.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
+
+# Peaq Network CharmEV Rust API Library
+
+#### Introduction
+This library handles the core api features of:
+* interacting with the node
+* connecting to provider peer network
```
Comment (faisalthaheem) [inline rust/README.md]: remove word network as it can be misleading

File: rust/README.md (hunk 1 above)
Comment (irediaes) [reply rust/README.md]: corrected

File: rust/README.md (hunk 2 above)
Comment (irediaes) [reply rust/README.md]: corrected

**Pull request merged**